src/py-opentimelineio/opentimelineio/media_linker.py

```diff
@@ -22,33 +22,37 @@
 # language governing permissions and limitations under the Apache License.
 #
 
-""" MediaLinker plugins fire after an adapter has read a file in order to
-produce MediaReferences that point at valid, site specific media.
+"""
+MediaLinker plugins fire after an adapter has read a file in order to
+produce :class:`.MediaReference`\s that point at valid, site specific media.
 
-They expose a "link_media_reference" function with the signature:
-link_media_reference :: otio.schema.Clip -> otio.core.MediaReference
+They expose a ``link_media_reference`` function with the signature:
 
-or:
-    def linked_media_reference(from_clip):
-        result = otio.core.MediaReference() # whichever subclass
-        # do stuff
-        return result
+.. py:function:: link_media_reference(in_clip: opentimelineio.schema.Clip) -> opentimelineio.core.MediaReference
+   :noindex:
+
+   Example link_media_reference function.
 
 To get context information, they can inspect the metadata on the clip and on
-the media reference.  The .parent() method can be used to find the containing
+the media reference. The :meth:`.Composable.parent` method can be used to find the containing
 track if metadata is stored there.
 
 Please raise an instance (or child instance) of
-otio.exceptions.CannotLinkMediaError() if there is a problem linking the media.
+:class:`.CannotLinkMediaError` if there is a problem linking the media.
```

Comment:
`CannotLinkMediaError` is not defined in this repo... What should we do?

Reply:
We can remove this sentence.